### PR TITLE
[Collection] Pass COLLECT_HUBS a/b test to context

### PR DESCRIPTION
**Pull request by damassi**
Companion PR to https://github.com/artsy/reaction/pull/2921

This updates the collect app to properly pass A/B test across to reaction code via context.

### CODE REVIEW

File: src/desktop/apps/collect/client.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ buildClientApp({
   context: {
     user: sd.CURRENT_USER,
     mediator,
+    COLLECTION_HUBS: sd.COLLECTION_HUBS,
```
Comment (damassi) [inline src/desktop/apps/collect/client.js]: We need to pass the A/B test over on both the client as well as the server. We can do this by injecting ENV var into context which we then pluck out via `SystemContext` in reaction.

File: src/desktop/apps/collect/server.tsx [hunk 2]
```diff
@@ -22,7 +22,9 @@ const index = async (req, res, next) => {
       routes: collectRoutes,
       url: req.url,
       userAgent: req.header("User-Agent"),
-      context: buildServerAppContext(req, res, { COLLECTION_HUBS }),
+      context: buildServerAppContext(req, res, {
+        COLLECTION_HUBS,
+      }),
```
Comment (damassi) [inline src/desktop/apps/collect/server.tsx]: Pass A/B test ENV var on server-side pass.

Comment (mbilokonsky) [reply src/desktop/apps/collect/server.tsx]: this one just looks like a formatting change?

Comment (damassi) [reply src/desktop/apps/collect/server.tsx]: Yah the only real change is in `client.tsx`

**Comment by pepopowitz** [comment]
It's strange to me that the only changes needed to make server-rendering properly function were to make changes in `client.js`. Can you go into more detail about why that's the case? It's opposite of what I would expect.

**Comment by damassi** [comment]
Yup! The issue was how we're accessing the prop in Reaction 👍

**Pull request merged**